biogrid/libs/biogrid-simulator/src/lib/biogrid/biogrid.spec.ts

```diff
@@ -69,18 +70,18 @@ describe('classes', () => {
     expect(positions.length).toEqual(2 + 6 + 4 + 1);
     expect(positions).toEqual([
       { x: 0, y: 0 },
-      { x: 5, y: 0.8333333333333333 },
+      { x: 5, y: 0.5 },
       { x: 5, y: 2.5 },
-      { x: 5, y: 4.166666666666667 },
-      { x: 5, y: 5.833333333333334 },
+      { x: 5, y: 4 },
+      { x: 5, y: 5.5 },
       { x: 5, y: 7.5 },
-      { x: 5, y: 9.166666666666666 },
-      { x: 5, y: 2.5 },
-      { x: 5, y: 7.5 },
-      { x: 5, y: 1.25 },
-      { x: 5, y: 3.75 },
-      { x: 5, y: 6.25 },
-      { x: 5, y: 8.75 },
+      { x: 5, y: 9 },
+      { x: 5.5, y: 2.5 },
+      { x: 5.5, y: 7.5 },
+      { x: 5, y: 1 },
+      { x: 5, y: 3.5 },
+      { x: 5, y: 6 },
+      { x: 5, y: 8.5 },
```

Comment:
can we shorten this test example at all in order to make it more readable + concise?